src/app/Components/sign-in/sign-in.component.ts

```diff
@@ -18,6 +19,13 @@ export class SignInComponent implements OnInit {
 
   ngOnInit(): void {}
 
+  seeError(e: Error): void {
+    this.loading = false;
```

Comment:
it would be better to keep `this.loading = false` in the same method where it was declared true